portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,33 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+var slideIndex = 1;
```

Comment:
Is there a way you can think of doing this without using a global variable?

Reply:
is there a reason this comment was resolved?

please don't resolve comments unless you address them by either making the change or comment as to why you think it is better to not change.

Reply:
I just saw a couple other javascript examples where this was done so I thought maybe it was more common with the language

Reply:
Could not find a way without the global variable and believe this is common practice. Swetha and I also discussed further in personal chat.